src/modules/toolbox/components/drawToolContent/DrawToolContent.js

```diff
@@ -12,6 +12,8 @@ import { AssetSourceType, getAssetSource, hexToRgb } from '../../../map/componen
 const Update = 'update';
 const Update_Tools = 'update_tools';
 const Update_FileSaveResult = 'update_fileSaveResult';
```

Comment:
signal for this type is never called. Can we remove it?

Reply:
No... a observer was mistakingly removed with the last commit. I'll fix that.